masterlist.yaml

```diff
@@ -4300,14 +4300,48 @@ plugins:
       - crc: 0xAD49CDCC
         util: 'FO4Edit v4.1.5k'
 
-  - name: 'WhoIsTheGeneral.esp'
+  - name: '(WhoIsTheGeneral|WTG( - |_)(CMRQL|CTI|KRIC|KCR|MRS|PGNRSQ|RQIC|RSFMQ|CMRQL - KRIC)( |_)Patch).esp'
```

Comment:
The period `.` before `esp` at the end should be escaped by writing a `\` in front of it.

The term
`(CMRQL|CTI|KRIC|KCR|MRS|PGNRSQ|RQIC|RSFMQ|CMRQL - KRIC)`

can also be written alphabetically and shortened to:
`(CMRQL( - KRIC)?|CTI|KCR|KRIC|MRS|PGNRSQ|RQIC|RSFMQ)`

Reply:
Is it a general style in the masterlist that i should sort regexes alphabetically instead of the order they are presented/released by the mod author?

And apologies for the obvious regex escaping error.

Reply:
I believe we do not have written this out somewhere in the documentation, so it's not a fixed rule. But at least in my opinion, it makes reading the Regex a bit easier, so - at least for new entries - I'd say we should write these alphabetically if possible.

Reply:
That seems like a good idea, and to add to it the order they are presented/released can always change, while the alphabet is unlikely to change any time soon.